scripts/src/java/org/oppia/android/scripts/common/remote/GitHubService.kt

```diff
@@ -29,7 +29,7 @@ interface GitHubService {
    * @return the list of [GitHubIssue]s read from the remote repository (as a [Call])
    */
   @Headers("Accept: application/vnd.github+json", "X-GitHub-Api-Version: 2022-11-28")
-  @GET("repos/{repo_owner}/{repo_name}/issues?direction=asc")
+  @GET("repos/{repo_owner}/{repo_name}/issues?direction=asc&pulls=false")
```

Comment:
Why not add the `pulls` as a query parameter? I think it is cleaner that way and will allow you to add tests in the corresponding test files. 

API ref: https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28

Reply:
As per doc , pulls  as Parameters work  for "List organization issues assigned to the authenticated user"

So, i have added new code , where i have filter out pulls using pull_request key.